ExerciseMVC/ExerciseMVC/Models/Task.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.ComponentModel.DataAnnotations;
+using System.Data.Entity;
+
+namespace ExerciseMVC.Models
+{
+    public class Task
+    {
+        public int TaskId { get; set; }
```

Comment:
As the model is itself a `Task`, would it have been better just to call this field `Id`?